conveyor/generator/generator/templates.go

```diff
@@ -0,0 +1,146 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package generator
+
+import (
+	"io"
+	"text/template"
+)
+
+var (
+	funcMap = template.FuncMap{
+		"inc": func(i int) int {
+			return i + 1
+		},
+		"notNil": func(x interface{}) bool {
+			return x == nil
+		},
+	}
+	stateMachineTpl = template.Must(template.New("stateMachineTpl").Funcs(funcMap).Parse(`
```

Comment:
Case when transition handler return error is not covered in template.
If it's not simple to do quickly, let's make all clear Transite handler return error

Reply:
I don't find in code any restrictions on return values of clear handlers.